### ISSUE TITLE
Minimum requirements for show? 

### ISSUE THREAD

**Issue by murchie85** [gayanvoice/most-active-github-users-nodejs#41]
Is there a min follower requirement? My profile murchie85 has over 3000 commits but not showing. kind regards

**Comment by gayanvoice** [comment]
Nope. App is not updated recently due to lack of time. I really need to add some changes. After that I think it will show up your profile. Actually it's not, there are no any other criteria.
